pkg/gds/admin.go

```diff
@@ -836,9 +834,9 @@ func (s *Admin) RetrieveVASP(c *gin.Context) {
 func (s *Admin) prepareVASPDetail(vasp *pb.VASP, log zerolog.Logger) (out *admin.RetrieveVASPReply, err error) {
 	// Create the response to send back
 	out = &admin.RetrieveVASPReply{
-		VerifiedContacts: models.VerifiedContacts(vasp),
-		Traveler:         models.IsTraveler(vasp),
+		Traveler: models.IsTraveler(vasp),
 	}
+	out.VerifiedContacts = models.VerifiedContacts(vasp)
```

Comment:
We could put this back into the struct now that we're changing the error handling.